tests/clusterclient.c

```diff
@@ -8,6 +8,10 @@
  *           Will send following commands using the `..ToNode()` API and a
  *           cluster node iterator to send each command to all known nodes.
  *
+ * Exit statuses this program can return:
+ *   0 - Successful execution of program.
+ *   1 - The required argument "HOST:PORT" is not given.
```

Comment:
Nice list!

Perhaps we should also do `exit(1)` when we print "Unknown argument", i.e. 1 = bad arguments.

Reply:
Yes, that probably good, Ill add that.